ui/src/flux/constants/functions.ts

```diff
@@ -15,12 +15,53 @@ export const FROM: FluxToolbarFunction = {
       type: 'String',
     },
   ],
+  package: '',
   desc:
     'Used to retrieve data from an InfluxDB data source. It returns a stream of tables from the specified bucket. Each unique series is contained within its own table. Each record in the table represents a single point in the series.',
   example: 'from(bucket: "telegraf/autogen")',
   category: 'Inputs',
   link:
-    'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/inputs/from',
+    'https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/inputs/from/',
```

Comment:
With this going into Chronograf, I assume these links should all point to the v1 docs. Just find and replace:

**Find:** `https://v2.docs.influxdata.com/v2.0/reference/flux/`
**Repl:** `https://docs.influxdata.com/flux/latest/`